config/ecs/config.go

```diff
@@ -2,12 +2,97 @@ package ecs
 
 import "github.com/crossplane/upjet/pkg/config"
 
+const Ecs = "ecs"
+
 // Configure configures individual resources by adding custom ResourceConfigurators.
 func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("alicloud_auto_provisioning_group", func(r *config.Resource) {
+		// We need to override the default group that upjet generated for
+		// this resource, which would be "ecs"
+		r.Kind = "AutoProvisioningGroup"
+		r.ShortGroup = Ecs
+	})
+	p.AddResourceConfigurator("alicloud_image", func(r *config.Resource) {
+		// We need to override the default group that upjet generated for
+		// this resource, which would be Ecs
+		r.ShortGroup = Ecs
+	})
+	p.AddResourceConfigurator("alicloud_image_copy", func(r *config.Resource) {
+		// We need to override the default group that upjet generated for
+		// this resource, which would be Ecs
+		r.Kind = "ImageCopy"
+		r.ShortGroup = Ecs
+	})
+	p.AddResourceConfigurator("alicloud_image_export", func(r *config.Resource) {
+		// We need to override the default group that upjet generated for
+		// this resource, which would be Ecs
+		r.Kind = "ImageExport"
+		r.ShortGroup = Ecs
+	})
+	p.AddResourceConfigurator("alicloud_image_import", func(r *config.Resource) {
+		// We need to override the default group that upjet generated for
+		// this resource, which would be Ecs
+		r.Kind = "ImageImport"
+		r.ShortGroup = Ecs
+	})
+	p.AddResourceConfigurator("alicloud_image_share_permission", func(r *config.Resource) {
+		// We need to override the default group that upjet generated for
+		// this resource, which would be Ecs
+		r.Kind = "ImageSharePermission"
+		r.ShortGroup = Ecs
+	})
+	p.AddResourceConfigurator("alicloud_instance", func(r *config.Resource) {
+		// We need to override the default group that upjet generated for
+		// this resource, which would be Ecs
+		r.ShortGroup = Ecs
```

Comment:
alicloud_instance depends on many resources, like vswitch, security_group, and please add references for it.